File: py/number3.py
def cnt_non_dec_series():
    len_series = 0
    cnt = 0

    length = int(input())

    if length > 0:
        num1 = int(input())
        num2 = 0

        for _ in range(1, length):
            num2 = int(input())
            if num2 >= num1:
                len_series += 1
            elif len_series != 0:
                cnt += 1
                len_series = 0
            num1 = num2

        if len_series != 0:
            cnt += 1
    return cnt


result = cnt_non_dec_series()
print(f"Количество неубывающих серий: {result}")

File: go/number3.go
package main

import "fmt"

func cntNonDecSeries() int {
	lenSeries := 0
	cnt := 0

	var length int
	fmt.Scanln(&length)

	if length > 0 {
		var num1 int
		fmt.Scanln(&num1)

		var num2 int
		for i := 1; i < length; i++ {
			fmt.Scanln(&num2)

			if num2 >= num1 {
				lenSeries++
			} else if lenSeries != 0 {
				cnt++
				lenSeries = 0
			}
			num1 = num2
		}

		if lenSeries != 0 {
			cnt++
		}
	}
	return cnt
}

func main() {
	result := cntNonDecSeries()
	fmt.Printf("Количество неубывающих серий: %d\n", result)
}